.brigade/brigade.ts

```diff
@@ -2,7 +2,7 @@ import { events, Event, Job, ConcurrentGroup, SerialGroup, Container } from "@br
 
 const azImg = "mcr.microsoft.com/azure-cli"
 const goImg = "brigadecore/go-tools:v0.6.0"
-const jsImg = "node:16.11.0-bullseye"
+const jsImg = "node:16.14.0-bullseye"
```

Comment:
Bumping to 16.4.0, but continuing to use Debian Bullseye for things like running linters and unit tests. This PR is really primarily concerned with changing the base images of images we _distribute_ as part of each Brigade release.